### ISSUE TITLE
Client/Server Configs applied to Common Only Config Entries

### ISSUE THREAD

**Issue by Speiger** [MehVahdJukaar/Supplementaries#655]
o/
You are using client sided configs on "fillItemCategory" which is something that can be called from both server/client sided.
Otherwise it would be marked with "OnlyIn(Dist.CLIENT)"

https://github.com/MehVahdJukaar/Supplementaries/blob/fb7073e114a8a94f166c28098b840b40d81d510f/src/main/java/net/mehvahdjukaar/supplementaries/common/items/BambooSpikesTippedItem.java#L79

It is causing this crash:
[crash-2023-06-15_03.34.50-server.txt](https://github.com/MehVahdJukaar/Supplementaries/files/11751685/crash-2023-06-15_03.34.50-server.txt)

Which was reported to me from someone who wanted to use these two mods together.
TL:DR client only configs should only affect the client :)

Anyways. If you need anymore infos i am glad to assist :)

**Comment by MehVahdJukaar** [comment]
hm thought i fixed this. are you using latest version?

**Comment by Speiger** [comment]
@MehVahdJukaar your code isn't up to date on Github.
The person who reported it. said it was fixed in the latest. But since the GH code still had it i reported it just to make sure it is.

**Comment by MehVahdJukaar** [comment]
hm i dont see that tho. if i go there it looks correct
https://github.com/MehVahdJukaar/Supplementaries/blob/master/common/src/main/java/net/mehvahdjukaar/supplementaries/common/items/BambooSpikesTippedItem.java

**Comment by Speiger** [comment]
@MehVahdJukaar https://github.com/MehVahdJukaar/Supplementaries/blob/1.19/src/main/java/net/mehvahdjukaar/supplementaries/common/items/BambooSpikesTippedItem.java
The 1.19 branch :)
Whatever master is :)

**Comment by MehVahdJukaar** [comment]
master is 1.19.2. 1.19 branch is that very old and buggy 1.19.0 beta port. i'll close as its solved
